src/gas_metering/mod.rs

```diff
@@ -480,6 +581,15 @@ fn add_grow_counter<R: Rules>(
 	b.build()
 }
 
+fn inject_counter<R: Rules>(
```

Comment:
what the reason to move this function?

Reply:
For readability reasons. See "Clean code" of Robert Martin. Or: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29 (Source code structure, rule 2).

Reply:
Ok, but it's not our code, so we must change nothing except what we really need to change.